src/PrestaShopBundle/Form/Admin/Improve/Design/Pages/CmsPageType.php

```diff
@@ -0,0 +1,279 @@
+                            'pattern' => '/^[^<>={}]*$/u',
+                            'message' => $this->translator->trans(
+                                '%s is invalid.',
+                                [],
+                                'Admin.Notifications.Error'
+                            ),
+                        ]),
+                        new Length([
+                            'max' => 255,
+                            'maxMessage' => $this->translator->trans(
+                                'This field cannot be longer than %limit% characters',
+                                ['%limit%' => 255],
+                                'Admin.Notifications.Error'
+                            ),
+                        ]),
+                    ],
+                ],
+            ])
+            ->add('meta_description', TranslatableType::class, [
+                'required' => false,
+                'options' => [
+                    'constraints' => [
+                        //@todo: typedRegexConstraint generic_name PR #12735
+                        new Regex([
+                            'pattern' => '/^[^<>={}]*$/u',
+                            'message' => $this->translator->trans(
+                                '%s is invalid.',
+                                [],
+                                'Admin.Notifications.Error'
+                            ),
+                        ]),
+                    ],
+                ],
+            ])
+            ->add('meta_keyword', TranslatableType::class, [
+                'type' => TextType::class,
+                'required' => false,
+                'options' => [
+                    'attr' => [
+                        'class' => 'js-taggable-field',
+                        'placeholder' => $this->translator->trans('Add tag', [], 'Admin.Actions'),
+                    ],
+                    'constraints' => [
+                        //@todo: typedRegexConstraint generic_name PR #12735
+                        new Regex([
+                            'pattern' => '/^[^<>={}]*$/u',
+                            'message' => $this->translator->trans(
+                                '%s is invalid.',
+                                [],
+                                'Admin.Notifications.Error'
+                            ),
+                        ]),
+                        new Length([
+                            'max' => 512,
+                            'maxMessage' => $this->translator->trans(
+                                'This field cannot be longer than %limit% characters',
+                                ['%limit%' => 512],
+                                'Admin.Notifications.Error'
+                            ),
+                        ]),
+                    ],
+                ],
+            ])
+            ->add('friendly_url', TranslatableType::class, [
+                'error_bubbling' => false,
```

Comment:
I'm wondering why some fields need `error_bubbling => false` and some others not, even though they also are `TranslatableType `

Reply:
@jolelievre that is currently required in TranslatableType when using constraint in parent level (not in options). If error bubbling isn't set to false, that parent error is shown somewhere in wrong level and is hidden by its child or something similar. Another case when using constraints in 'options' works well with default error bubbling.

Reply:
wow that's tricky 🤪
and it's not possible to define all the constraints on the same level? either only parent or only in the options? (I guess there's a reason)
I'm wondering if this could be managed automatically by `TranslatableType` in its resolve function, something like
```php
if (!isset($options['error_bubbling']) && !empty($options['constraints']) && !empty($options['options']['constraints'])) {
    $options['error_bubbling'] = false;
}
```

I'm just proposing this because who is not aware of this particularity has great chances to forget it.

Reply:
@jolelievre  Maybe worth creating an issue to fix that confusion in some way ? (here this will definitely be lost in minutes :D )

Reply:
Yeah of course I didn't plan to modify this in this PR ^^

Reply:
I created this issue for later https://github.com/PrestaShop/PrestaShop/issues/13107